services/aur/aur.tester.js

```diff
@@ -14,7 +14,7 @@ const t = (module.exports = new ServiceTester({
 // version tests
 
 t.create('version (valid)')
-  .get('/version/dropbox.json')
+  .get('/version/visual-studio-code-bin.json')
```

Comment:
Not _directly_ related to the core bux fix in this PR, but this test was failing for me locally (apparently the version for the dropbox package is now "out of date" which was resulting in the color being `orange`) so I went ahead and fixed this too.